src/internal/packager2/find_images.go

```diff
@@ -464,6 +472,23 @@ func appendToImageMap(imgMap map[string]bool, pod corev1.PodSpec) map[string]boo
 	return imgMap
 }
 
+func appendToImageMapOCIRepo(ctx context.Context, imgMap map[string]bool, repo sourcev1beta2.OCIRepositorySpec) map[string]bool {
+	var url = strings.TrimPrefix(repo.URL, "oci://")
+
+	if repo.Reference.Tag != "" {
+		url = url + ":" + repo.Reference.Tag
+	} else if repo.Reference.Digest != "" {
+		url = url + "@" + repo.Reference.Digest
+	} else if repo.Reference.SemVer != "" || repo.Reference.SemverFilter != "" {
+		logger.From(ctx).Error("can not use semver or semverFilter with OCIRepository")
```

Comment:
We should use the `Warn` function rather than the `Error` function. We should also give users the image url, and oci repository name so they can easily find it. (note: you'll have to pass in `sourcev1beta2.OCIRepository` for this) 
```suggestion
		logger.From(ctx).Warn("cannot create image reference with semver or semverFilter", "image", url, "OCIRepository", repo.Name)
```

Reply:
![image](https://github.com/user-attachments/assets/b13deff6-e22a-41ce-8afb-3f085df1d14a)

Reply:
Updated PR with your comments Austin